api/CHANGELOG.md

```
@@ -22,6 +22,10 @@ Each entry must include the Github issue reference in the following format:
 
 ## [Unreleased]
 
+# [v1.0.0]()
```

markdownlint-cli2 (0.17.2)

No empty links
null

(MD042, no-empty-links)